src/serac/coefficients/coefficient_extensions.hpp

```diff
@@ -571,6 +584,13 @@ class SurfaceElementAttrCoefficient : public mfem::Coefficient {
    */
   SurfaceElementAttrCoefficient(mfem::ParMesh& mesh, mfem::Coefficient& c) : coef_(c), pmesh_(mesh) {}
 
+  /**
+   * @brief Evaluates an element attribute-based coefficient on the a boundary element
+   *
+   * @param[in] Tr The local FE transformation
```

Comment:
Thanks @white238 for fixing a lot of these.
```suggestion
   * @param[in] Tr The local surface FE transformation
```